ArduPlane/quadplane.cpp

```diff
@@ -1907,15 +1923,19 @@ void QuadPlane::update_transition(void)
         // calculate transition rate in degrees per
         // millisecond. Assume we want to get to the transition angle
         // in half the transition time
-        float transition_rate = tailsitter.transition_angle / float(transition_time_ms/2);
+        float transition_rate = tailsitter.transition_angle_fw / float(transition_time_ms/2);
         uint32_t dt = now - transition_start_ms;
-        plane.nav_pitch_cd = constrain_float(transition_initial_pitch - (transition_rate * dt)*100, -8500, 8500);
+        float pitch_cd;
+        pitch_cd = constrain_float((-transition_rate * dt) * 100, -8500, 0);
+        // if already pitched forward at start of transition, wait until curve catches up
+        plane.nav_pitch_cd = (pitch_cd > transition_initial_pitch) ? transition_initial_pitch : pitch_cd;
```

Comment:
This is a regression on https://github.com/ArduPilot/ardupilot/pull/16294

Reply:
yep...his stuff was before that...will fix